src/theme/components-overrides.scss

```diff
@@ -223,3 +223,23 @@ $types: 'error', 'info', 'success', 'warning';
     border-bottom-color: var(--bcgov-notification-border-#{$notificationType});
     border-right-color: var(--bcgov-notification-border-#{$notificationType})}
 }
+
+.card-overflow{
+  border-radius: 4px;
+}
```

Comment:
If this rule is specific only to the card component, it should be moved to the component style file.